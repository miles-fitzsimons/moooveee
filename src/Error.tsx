import React from "react";

function Error() {
  return (
    <div className="error">
      <h1>Something went wrong</h1>
      <h2>Maybe try refreshing the page</h2>
    </div>
  );
}

export default Error;
