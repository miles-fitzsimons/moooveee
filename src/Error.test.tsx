import React from "react";
import { shallow } from "enzyme";
import Error from "./Error";

it("renders correctly", () => {
  const component = shallow(<Error />);
  expect(component).toMatchSnapshot();
});
