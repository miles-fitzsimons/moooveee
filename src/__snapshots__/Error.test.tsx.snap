// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="error"
>
  <h1>
    Something went wrong
  </h1>
  <h2>
    Maybe try refreshing the page
  </h2>
</div>
`;
